src/db/migrations/1733118742233-AddNumberOfStaffToUniversity.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner, TableColumn } from 'typeorm';
+
+export class AddNumberOfStaffToUniversity1733118742233 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.addColumn('universities', new TableColumn({
+      name: 'number_of_staff',
+      type: 'int',
+      isNullable: true,
```

Comment:
Consider adding a default value for the 'number_of_staff' column to ensure data consistency and avoid potential null-related issues.